### DISCUSSION TITLE
It look like the installer schema need some update.

### DISCUSSION THREAD

**Discussion by BrandonWanHuanSheng** [microsoft/winget-pkgs#107504] [Q&A] [answered]
Where is the PackageIdentifier of the PackageDependencies:? It look like it was not in the schema https://aka.ms/winget-manifest.installer.1.4.0.schema.json.

**Comment by Trenly** [comment]
> Where is the PackageIdentifier of the PackageDependencies:? It look like it was not in the schema [aka.ms/winget-manifest.installer.1.4.0.schema.json](https://aka.ms/winget-manifest.installer.1.4.0.schema.json).

It's included - https://github.com/microsoft/winget-cli/blob/1cc043b541fff91f58065b8e015aeedde356aa01/schemas/JSON/manifests/v1.4.0/manifest.installer.1.4.0.json#L319-L331

**Reply by BrandonWanHuanSheng** [reply]
But it did not show up on wingetcreate instead of MinimumVersion

**Reply by mdanish-kh** [reply, marked as answer]
wingetcreate would likely add support for Dependencies when they are fully supported by the WinGet client.

- https://github.com/microsoft/winget-cli/issues/163
